packages/nimble-components/src/select/specs/built-in-labels.md

```diff
@@ -0,0 +1,77 @@
+# Built-in Labels for Select and Combobox
+
+## Problem Statement
+
+Most Nimble controls offer built in labels but the select and combobox controls don't, meaning clients have had to create and style labels manually.
+
+Built-in labels will reduce the effort needed for clients to consume these controls, as well as resolving some accessibility issues (see work item links below).
+
+## Links To Relevant Work Items and Reference Material
+
+[Built-in label for select and combobox](https://github.com/ni/nimble/issues/2183)
+
+Related issues:
+
+-   [Resolve label accessibility issues](https://github.com/ni/nimble/issues/94)
+-   [Fix accessibility issues in Angular Example app](https://github.com/ni/nimble/issues/280)
+
+## Implementation / Design
+
+Implementation details apply to both `nimble-select` and `nimble-combobox`, unless otherwise specified.
+
+### API (Slots)
+
+Default Slot:
+
+-   The `nimble-list-option`s in the control are placed in this slot currently.
+-   We will add support for specifying the label as text content in the same slot. For example:
+    ```html
+    <nimble-select>
+        Select Label Text
```

Comment:
We'll need to support putting the label in a `<span>` in the default slot too. This would allow Angular clients to set an `i18n` tag on it to trigger its content to be localized. Without that they'd have to put the tag on the `<nimble-select>` which would trigger the HTML tags of the options to be localized too.

You can see an example of this in the [docs for the list option group default slot](https://nimble.ni.dev/storybook/?path=/docs/components-select--docs#list-option-group).

Reply:
Good point, added that detail.